rustls/src/crypto/ring/kx.rs

```diff
@@ -76,22 +91,38 @@ pub static X25519: &dyn SupportedKxGroup = &KxGroup {
     //  are not compliant to SP 800-56Arev3."
     // -- <https://csrc.nist.gov/csrc/media/Projects/cryptographic-module-validation-program/documents/fips%20140-3/FIPS%20140-3%20IG.pdf>
     fips_allowed: false,
+
+    pub_key_validator: x25519_validator,
 };
 
+fn x25519_validator(point: &[u8]) -> bool {
+    point.len() == 32
+}
```

Comment:
This `fn` feels small enough that maybe it could just be in-lined into the `pub_key_valdiator` field initialization as a closure? (If you agree, ditto for `SECP256R1` & `SECP384R1`).

Reply:
I have inlined the x25519 one, and kept the other one separate so it has a name and the citation has one place to live.